test/src/test/java/org/corfudb/integration/CorfuReplicationE2EIT.java

```diff
@@ -25,7 +25,8 @@ public static Collection input() {
 
         List<String> transportPlugins = Arrays.asList(
                 "src/test/resources/transport/grpcConfig.properties",
-                "src/test/resources/transport/nettyConfig.properties");
+                "src/test/resources/transport/nettyConfig.properties",
```

Comment:
no need for a new properties file.  We are only testing dynamic mode now.  You can reuse nettyConfig.properties.

It will in turn have the effect of all existing tests discovering streams dynamically.